test/irmin-pack/dune

```diff
@@ -1,24 +1,7 @@
-(library
- (name test_pack)
- (modules test_pack multiple_instances test_existing_stores layered
-   test_inode import)
- (libraries alcotest fmt common index irmin irmin-test irmin-pack
-   irmin-pack.layered logs lwt lwt.unix fpath)
- (preprocess
-  (pps ppx_irmin)))
-
-(executable
+(test
  (name test)
- (modules test test_utils)
- (libraries alcotest astring irmin irmin-test test_pack))
-
-(rule
- (alias runtest)
  (package irmin-pack)
- (action
-  (run ./test.exe -q --color=always)))
```

Comment:
From what @zshipko describes in https://github.com/mirage/irmin/pull/1313#issuecomment-818840782, the slow tests for `irmin-pack` take many minutes to run (which is fine for CI). What's the new time for a fresh `dune runtest` with these changes?

Reply:
Right, I'm super confused, for some reason I though that `-q` was the default. Would be nice if we could set this up in the test files - but in any case I'm happy to close the PR as this does indeed seems to be a regression.

Reply:
(and that we were doing this for `--color=always` which is now fixed in alcotest)

Reply:
@CraigFe I actually disabled the longer running test of the two: https://github.com/mirage/irmin/blob/94ccbd2d64eb32f515f577ae58948fa06b76038b/src/irmin-test/store.ml#L2048